skyfield/tests/test_stars.py

```diff
@@ -1,8 +1,25 @@
+from pyongc import data
 from skyfield import api
-from skyfield.data.hipparcos import load_dataframe
+from skyfield.data import hipparcos, ongc
 
 def test_dataframe():
     with api.load.open('hip_main.dat.gz') as f:
-        df = load_dataframe(f)
+        df = hipparcos.load_dataframe(f)
     star = api.Star.from_dataframe(df)
     assert repr(star) == 'Star(ra shape=9933, dec shape=9933, ra_mas_per_year shape=9933, dec_mas_per_year shape=9933, parallax_mas shape=9933, epoch shape=9933)'
+
+def test_dataframe_from_ongc():
```

Comment:
Thank you for including tests! Many developers don't think to write them, and it's very helpful.